server/routes/manageConditions/handlers/outofboundsPremises/outOfBoundsPremisesRemovalRoutes.ts

```diff
@@ -0,0 +1,62 @@
+import { Request, Response } from 'express'
+import LicenceService from '../../../../services/licenceService'
+import YesOrNo from '../../../../enumeration/yesOrNo'
+import { AdditionalCondition, Licence } from '../../../../@types/licenceApiClientTypes'
+
+export default class OutOfBoundsPremisesRemovalRoutes {
+  constructor(private readonly licenceService: LicenceService) {}
+
+  private getLicenceCondition = (conditionId: number, licence: Licence): AdditionalCondition | null => {
+    const condition = licence.additionalLicenceConditions.find((c: AdditionalCondition) => c.id === conditionId)
+    return condition || null
+  }
+
+  GET = async (req: Request, res: Response): Promise<void> => {
+    const { licence } = res.locals
+    const { conditionId } = req.params
+
+    const condition = this.getLicenceCondition(parseInt(conditionId, 10), licence)
+
+    let description = ''
+    if (condition.data && condition.data.length) {
+      description = condition.data[0].value
+      if (condition.data[0].field === 'premisesAddress') {
+        ;[description] = condition.data[0].value.split(',')
+      }
```

Comment:
Not sure if this is better but... 
```js
    let description = ''
    if (condition.data && condition.data.length) {
      description = condition.data[0].value
      if (condition.data[0].field === 'premisesAddress') {
        ;[description] = condition.data[0].value.split(',')
      }
```
=>
```js
    let description = ''
    if (condition?.data.length) {
      const {field, value} = condition.data[0]
      description =  field !== 'premisesAddress' ?  value : value.split(',')[0]
```

Reply:
yes much better, updated